test/common/tessel-simulator.js

```diff
@@ -7,6 +7,7 @@ function TesselSimulator() {
 
   tessel.connection = {
     exec: function(command, callback) {
+      tessel._rps.control.write(command.join(' '));
```

Comment:
This will blow up if command is not an array. I suggest either: 

- Check if it's an array and if not, make it an empty array, or...
- Throw if it's likely to put the program in an unknown state